File: MODUL 4/PRAK404-2410817210005-Muhammad_Irgi_Fahrezha.py
while True:
    print("\nPilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan : "))


    if pilihan == 1:
        nilai_pertama = float(input("Masukkan nilai pertama :"))
        nilai_kedua = float(input("Masukkan nilai kedua :"))
        hasil = nilai_pertama + nilai_kedua
        print(f"Hasil penjumlahan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")

    elif pilihan == 2:
        nilai_pertama = float(input("Masukkan nilai pertama :"))
        nilai_kedua = float(input("Masukkan nilai kedua :"))
        hasil = nilai_pertama - nilai_kedua
        print(f"Hasil pengurangan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")

    elif pilihan == 3:
        nilai_pertama = float(input("Masukkan nilai pertama :"))
        nilai_kedua = float(input("Masukkan nilai kedua :"))
        hasil = nilai_pertama * nilai_kedua
        print(f"Hasil perkalian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")

    elif pilihan == 4:
        nilai_pertama = float(input("Masukkan nilai pertama :"))
        nilai_kedua = float(input("Masukkan nilai kedua :"))
        hasil = nilai_pertama / nilai_kedua
        print(f"Hasil pembagian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")

    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Irgi Fahrezha")
        break
        
    else:
        print("Input anda salah, silahkan coba lagi")

File: MODUL 4/PRAK404-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int main () {

    int pilihan;
    float nilai_pertama;
    float nilai_kedua;
    float hasil;

    do {
        printf("\nPilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);

        switch (pilihan) {
            case 1:
                printf("Masukkan nilai pertama : ");
                scanf("%f", &nilai_pertama);
                printf("Masukkan nilai kedua : ");
                scanf("%f", &nilai_kedua);
            
                hasil = nilai_pertama + nilai_kedua;

                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;

            case 2:
            printf("Masukkan nilai pertama : ");
                scanf("%f", &nilai_pertama);
                printf("Masukkan nilai kedua : ");
                scanf("%f", &nilai_kedua);
                
                hasil = nilai_pertama - nilai_kedua;

                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;

            case 3:
                printf("Masukkan nilai pertama : ");
                scanf("%f", &nilai_pertama);
                printf("Masukkan nilai kedua : ");
                scanf("%f", &nilai_kedua);
                
                hasil = nilai_pertama * nilai_kedua;

                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;

            case 4:
                printf("Masukkan nilai pertama : ");
                scanf("%f", &nilai_pertama);
                printf("Masukkan nilai kedua : ");
                scanf("%f", &nilai_kedua);
                
                hasil = nilai_pertama / nilai_kedua;

                printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
                break;
            case 5:
                printf("Terimakasih, telah menggunakan kalkulator Muhammad Irgi Fahrezha\n");
                break;

            default:
                printf("Input anda salah, silahkan coba lagi\n");
                break;
        }
    }
    while (pilihan != 5);
    
    return 0;

}